api/models/Prebooking.js

```diff
@@ -10,8 +10,15 @@ var ModelHelpers = require('../lib/ModelHelpers');
 var setNormalisedRelationships = (record, done) => {
   delete record.male_prebooking;
   delete record.female_prebooking;
+  delete record.male_contingency;
+  delete record.female_contingency;
+
+  if (Boolean(record.contingency)) {
```

Comment:
imho it should be `if (record.contingency === true)` which is the sort of pattern we use elsewhere